packages/grafana-ui/src/components/Forms/Form.mdx

```diff
@@ -182,6 +185,92 @@ const defaultValues: FormDto {
 </Forms.Form>
 ```
 
+### Validation
+
+Validation can be performed either synchronously or asynchronously. What's important here is that the validation function must return either a `boolean` or a `string`.
+
+#### Basic required example
+
+```jsx
+<Forms.Form ...>{
+  ({register, errors}) => (
+    <>
+      <Forms.Field invalid={!!errors.name} error={errors.name && 'Name is required'}
```

Comment:
This can be also simplified to `error={errors.name?.message}` and the message is then passed to `required` in `register`.